homeassistant/components/ebox/sensor.py

```diff
@@ -96,9 +156,10 @@ def __init__(self, ebox_data, sensor_type, name):
         """Initialize the sensor."""
         self.client_name = name
         self.type = sensor_type
-        self._name = SENSOR_TYPES[sensor_type][0]
-        self._unit_of_measurement = SENSOR_TYPES[sensor_type][1]
-        self._icon = SENSOR_TYPES[sensor_type][2]
+        metadata = SENSOR_TYPES[sensor_type]
+        self._name = metadata.name
+        self._unit = metadata.unit
+        self._icon = metadata.icon
```

Comment:
Why not use _attr_ variables right away?
```suggestion
        self._attr_unit_of_measurement = metadata.unit
        self._attr_icon = metadata.icon
```

Reply:
Very elegant idea, DONE. Even though I am no fan of having _ variables that are used outside the class (in this case declared in super). There should have been a standard like __variable for variables used with inheritance.